edumanage/migrations/0012_instserver_psk.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2025-01-08 11:14
```

Comment:
Given that #109 got merged in the meantime, perhaps the migration here should be renamed to `0013` ?

Reply:
rebased to include #109 and fixed